setup/README.md

```diff
@@ -0,0 +1,12 @@
+# Cloud 9
```

Comment:
Thanks for submitting this; I think it could be very helpful. Can I suggest adding a sentence saying *what Cloud9 is*, and how it differs from other setup options?

This demo app is evolving into something that could be either a template for COS projects, or an external tool- specifically because it will contain both a docker file and Cloud9 scripts, I think it might help new users to know what this folder is, and how it fits into other deployment/usage options.